### PR TITLE
Bugfix required checks

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/app/controllers/status/required_checks_controller.rb [hunk 1]
```diff
@@ -22,11 +22,12 @@ def index
     @required_checks = @checkable.required_checks
   end
 
-  # POST /status_reports/projects/:project_name/required_checks/:name
-  # POST /status_reports/repositories/:project_name/:repository_name/required_checks/:name
+  # POST /status_reports/projects/:project_name/required_checks
+  # POST /status_reports/repositories/:project_name/:repository_name/required_checks
   def create
     authorize @checkable
-    @required_checks = @checkable.required_checks |= names
```
Comment (hennevogel) [inline src/api/app/controllers/status/required_checks_controller.rb]: I'm still a bit puzzled how the tests ever passed this line

```
irb(main):001:0> [] |= ['one', 'two']
Traceback (most recent call last):
        1: from /home/hvogel/.orr/bin/irb:11:in `<main>'
SyntaxError ((irb):1: syntax error, unexpected tOP_ASGN, expecting end-of-input
[] |= ['one', 'two']
   ^~)
irb(main):002:0> ['one'] |= ['one', 'two']
Traceback (most recent call last):
        1: from /home/hvogel/.orr/bin/irb:11:in `<main>'
SyntaxError ((irb):2: syntax error, unexpected tOP_ASGN, expecting end-of-input
['one'] |= ['one', 'two']
        ^~)
```

File: src/api/spec/controllers/status/required_checks_controller_spec.rb [hunk 2]
```diff
@@ -60,6 +63,15 @@
       it { expect(response).to have_http_status(:success) }
```
Comment (hennevogel) [inline src/api/spec/controllers/status/required_checks_controller_spec.rb]: My suspicion is that it is somehow this line. No idea what `it` is. It is not the `post` from above.

Comment (bgeuken) [reply src/api/spec/controllers/status/required_checks_controller_spec.rb]: The request is missing here.

File: src/api/app/controllers/status/required_checks_controller.rb (hunk 1 above)
Comment (ChrisBr) [reply src/api/app/controllers/status/required_checks_controller.rb]: You need to assign it to a variable:

```
tmp = []
tmp |= ["one"]
# => ["one"]
```

**Comment by ChrisBr** [comment]
Closing for now as the code works as expected.

The only thing what is not working is testing the http_status as this happens in the block of the previous it statement. We need to fix this in another PR.

**Pull request closed without merging**